solar-system-project.rb

```diff
@@ -0,0 +1,380 @@
+    # in m/s^2
+    gravity: 3.7,
+    #in K
+    avg_temp: 340,
+    # million mi
+    distance_from_sun: 35.98,
+  },
+  {
+    name: "venus",
+    position: 2,
+    # in km
+    radius: 6051.8,
+    # in kg
+    mass: "4.6023 x 10^8",
+    # in m/s^2
+    gravity: 8.87,
+    #in K
+    avg_temp: 737,
+    # million mi
+    distance_from_sun: 67.24,
+  },
+  {
+    name: "earth",
+    position: 3,
+    # in km
+    radius: 6371.0,
+    # in kg
+    mass: "5.97237 x 10^24",
+    # in m/s^2
+    gravity: 9.807,
+    #in K
+    avg_temp: 288,
+    # million mi
+    distance_from_sun: 92.96,
+  },
+  {
+    name: "mars",
+    position: 4,
+    # in km
+    radius: 3389.5,
+    # in kg
+    mass: "6.4171 x 10^23",
+    # in m/s^2
+    gravity: 3.711,
+    #in K
+    avg_temp: 210,
+    # million mi
+    distance_from_sun: 141.6,
+  },
+  {
+    name: "jupiter",
+    position: 5,
+    # in km
+    radius: 69911,
+    # in kg
+    mass: "1.8982 x 10^27",
+    # in m/s^2
+    gravity: 24.79,
+    #in K
+    avg_temp: 165,
+    # million mi
+    distance_from_sun: 438.8,
+  },
+  {
+    name: "saturn",
+    position: 6,
+    # in km
+    radius: 58232,
+    # in kg
+    mass: "5.6834 x 10^26",
+    # in m/s^2
+    gravity: 10.44,
+    #in K
+    avg_temp: 134,
+    # million mi
+    distance_from_sun: 888.2,
+  },
+  {
+    name: "uranus",
+    position: 7,
+    # in km
+    radius: 25362,
+    # in kg
+    mass: "8.6810 x 10^25",
+    # in m/s^2
+    gravity: 8.69,
+    #in K
+    avg_temp: 76,
+    # million mi
+    distance_from_sun: 1.784,
+  },
+  {
+    name: "neptune",
+    position: 8,
+    # in km
+    radius: 24622,
+    # in kg
+    mass: "1.0243 x 10^26",
+    # in m/s^2
+    gravity: 11.15,
+    #in K
+    avg_temp: 72,
+    # million mi
+    distance_from_sun: 2795,
+  }
+]
+
+### END ARRAY OF HASHES WITH BASIC INFO ABT PLANETS ###
+### START DEFINITION OF SOLAR SYSTEM CLASS ###
+
+# SolarSystem class makes it possible to add another planet, return planets with numbers, and print everything
+class SolarSystem
+
+  attr_accessor :planets
+
+  def initialize(planet_instances)
+    @planets = planet_instances
+  end
+
+  def add_another_planet(user_input)
+    @planets << Planet.new(user_input)
+  end
+
+  def return_planets_with_numbers
+    planets_with_numbers = []
+    @planets.each_with_index do |planet, index|
+      planets_with_numbers << "#{index + 1}. #{planet.name.capitalize}"
+    end
+    return planets_with_numbers
+  end
+
+  def print_everything
+    entire_encyclopedia = []
+    @planets.each_with_index do |planet, index|
+    entire_encyclopedia << planet.planet_info
+    end
+    return entire_encyclopedia
+  end
+
+end
+
+### END DEFINITION OF SOLAR SYSTEM CLASS ###
+### START DEFINITION OF PLANET CLASS ###
+
+# Planet class makes it possible to store information about a planet and print the corresponding details
+class Planet
+
+  attr_accessor :name :position, :radius, :mass, :gravity, :avg_temp, :distance_from_sun
```

Comment:
Line 160 gave me a syntax error, but with a comma between `:name` and `:position` everything ran fine.